src/plugins/cookies/index.js

```diff
@@ -0,0 +1,150 @@
+import _ from 'lodash';
+import URI from 'URIjs';
+import ChromeCookie from './cookie';
+import ChromeEvent from '../../events';
+import { assertGet, assertGetAll, assertSet, assertRemove } from './assert';
+
+export default class ChromeCookies {
+
+    constructor (state = []) {
+        this._state = state;
+        this.onChanged = new ChromeEvent();
```

Comment:
It's native chrome event name
https://developer.chrome.com/extensions/cookies#event-onChanged